spikesorters/ironclust/mdarecordingextractor2.py

```diff
@@ -0,0 +1,162 @@
+import json
+import numpy as np
+import os
+
+from spikeextractors import RecordingExtractor
+from spikeextractors import SortingExtractor
+
+from .mdaio import DiskReadMda, readmda, writemda32, writemda64, writemda
+
+
+class MdaRecordingExtractor2(RecordingExtractor):
```

Comment:
Why not impplementing this extractor in spikeextractors ?

Reply:
We could... we should! I just didn't want to modify too much at once.

Reply:
I think we could just give optional argument to the MdaRecordingExtractor :)